client/components/LearnerViewDirective.js

```diff
@@ -239,6 +318,7 @@ tie.directive('learnerView', [function() {
           $scope.previousInstructions = [];
           $scope.nextButtonIsShown = false;
           $scope.feedbackParagraphs = introParagraphs;
+          $scope.unlockedQuestions[$scope.currentQuestionIndex] = true;
```

Comment:
I could do this...

        $scope.questionIds = questionSet.getQuestionIds();
        $scope.unlockedQuestions = [];
        for (var i = 1; i < $scope.questionIds.length; i++) {
          $scope.unlockedQuestions[i] = false;
        }

Something like that? Any better way to do this?